kolibri/plugins/coach/serializers.py

```diff
@@ -68,12 +67,11 @@ def get_last_active(self, target_user):
 class ContentReportSerializer(serializers.ModelSerializer):
     progress = serializers.SerializerMethodField()
     last_active = serializers.SerializerMethodField()
-    parent = serializers.SerializerMethodField()
 
     class Meta:
         model = ContentNode
         fields = (
-            'pk', 'content_id', 'title', 'progress', 'kind', 'last_active', 'parent',
+            'pk', 'content_id', 'title', 'progress', 'kind', 'last_active',
```

Comment:
It wouldn't be terrible to reinstate just as long as we did it with the foreign key rather than the `target_node.get_ancestors().last()` call that was being done, which was yuuuuugely inefficient.